src/webob/acceptparse.py

```diff
@@ -358,6 +381,9 @@ def _python_value_to_header_str(cls, value):
         Convert Python value to header string for __add__/__radd__.
```

Comment:
Both the class method's name and its docstring are saying that the class method returns a header string, but it has been modified to return `None` when `value` is `None`?

The class method is also used in `.__init__` and `accept_property` (and indirectly in `.__add__` and `.__radd__`, via use of `create_accept_header`), so the docstring saying that it is for `__add__` and `__radd__` is no longer true?